backend/src/vscode-youi-events.ts

```diff
@@ -46,21 +46,22 @@ export class VSCodeYouiEvents implements YouiEvents {
             });
             return "installing_dependencies_completed";
         });
-        return;
     }
 
-    private showDoneMessage(targetPath: string): void {
+    private showDoneMessage(success: boolean, message: string, targetPath: string): void {
         VSCodeYouiEvents.installing = false;
         const OpenWorkspace = 'Open Workspace';
-        const commandName_OpenWorkspace = "vscode.openFolder";
-        const commandParam_OpenWorkspace = targetPath;
-        vscode.window.showInformationMessage('The project has been successfully generated.\nWould you like to open it?', OpenWorkspace)
-            .then(selection => {
+        const commandNameOpenWorkspace = "vscode.openFolder";
+        const commandParamOpenWorkspace = targetPath;
```

Comment:
not a must but could be inside the if {}

Reply:
Done